test/test_openmm_amber.py

```diff
@@ -1713,12 +1713,12 @@ class TestAmoebaParm(TestCaseRelative):
     def test_amoeba_forces(self):
```

Comment:
Can you remove the `@pytest.mark.xfail` line above the `TestAmoebaParm` class?  That line makes failed tests in this test class non-fatal (in the test report, it marks tests that fail with an `x` but still reports a successful run, or with a capital `X` if the test actually passed).

https://docs.pytest.org/en/latest/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail